### PR TITLE
Berlin-veganify Qt app

### PR DESCRIPTION
**Pull request by jmastr**

### CODE REVIEW

File: qml/Silica4v-play/qmldir [hunk 1]
```diff
@@ -10,7 +10,6 @@ Label 1.0 Label.qml
 ListItem 1.0 ListItem.qml
 MenuItem 1.0 MenuItem.qml
 OpacityRampEffect 1.0 OpacityRampEffect.qml
-Page 1.0 Page.qml
```
Comment (micuintus) [inline qml/Silica4v-play/qmldir]: With every component we move from the Silica wrapper to our own component abstraction, I wonder whether we actually want to stick with this hybrid approach or whether we maybe should move everything from ```Silica4v-play``` to ```components```. But I am not sure. Lets stick with the hybrid approach for now (as it reduces the amount of empty components just sitting there to wrap some different names) and see how we proceed. K?

File: qml/pages/VenueList.qml [hunk 2]
```diff
@@ -137,6 +137,9 @@ Page {
 
             onClicked:
             {
+                // ios: keyboard stays visible, if user used search field before clicking and did not press Return key
+                Qt.inputMethod.hide();
```
Comment (micuintus) [inline qml/pages/VenueList.qml]: Interesting!

File: qml/Silica4v-play/qml/config.json [hunk 3]
```diff
@@ -0,0 +1,8 @@
+{
+  "title": "Berlin Vegan",
+  "identifier": "com.jmastr.harbour-berlin-vegan",
```
Comment (micuintus) [inline qml/Silica4v-play/qml/config.json]: Hilarious, the mixture of ```harbour``` and ```jmastr```!

File: android/AndroidManifest.xml [hunk 4]
```diff
@@ -0,0 +1,60 @@
...
+            <!-- Run with local libs -->
+            <meta-data android:name="android.app.use_local_qt_libs" android:value="-- %%USE_LOCAL_QT_LIBS%% --"/>
+            <meta-data android:name="android.app.libs_prefix" android:value="/data/local/tmp/qt/"/>
+            <meta-data android:name="android.app.load_local_libs" android:value="-- %%INSERT_LOCAL_LIBS%% --"/>
+            <meta-data android:name="android.app.load_local_jars" android:value="-- %%INSERT_LOCAL_JARS%% --"/>
+            <meta-data android:name="android.app.static_init_classes" android:value="-- %%INSERT_INIT_CLASSES%% --"/>
+            <!--  Messages maps -->
+            <meta-data android:value="@string/ministro_not_found_msg" android:name="android.app.ministro_not_found_msg"/>
+            <meta-data android:value="@string/ministro_needed_msg" android:name="android.app.ministro_needed_msg"/>
+            <meta-data android:value="@string/fatal_error_msg" android:name="android.app.fatal_error_msg"/>
```
Comment (micuintus) [inline android/AndroidManifest.xml]: Oh man, Android, so much noise. Especially in these XML files...

File: qml/Silica4v-play/Separator.qml [hunk 5]
```diff
@@ -4,4 +4,7 @@ Rectangle {
 
     property color color
     property var horizontalAlignment
+
+    visible: false
```
Comment (micuintus) [inline qml/Silica4v-play/Separator.qml]: BTW: Is it our plan to keep it like that, i.e. never show this separator?

**Comment by micuintus** [comment]
I am just amazed how you made it possible to make the app look so much closer as the native Android app without even touching the generic files*! Starts looking pretty good on v-play.
HOT STUFF! ;) 

# 🥇 

*) BTW: I wonder whether we also would like to bring the SailfishOS app closer to that design (without the app feeling not native on Sailfish anymore or looking weird). But maybe we do not even want to do this? I don't know yet. To a certain degree I'd also like to leave the Sailifish variant looking different, as it demonstrates how frickin' awesome Qt, Silica and v-play are and what we have made out of it.

File: qml/Silica4v-play/Separator.qml (hunk 5 above)
Comment (jmastr) [inline qml/Silica4v-play/Separator.qml]: Don't know yet. We need a design-concept soonish! ;)

**Pull request merged**